components/accept-financial-contributions/HostCollectiveCard.js

```diff
@@ -111,15 +120,15 @@ const HostCollectiveCard = ({ host, collective, onChange, ...props }) => {
         <Container pl={3} flexShrink={1}>
           <Flex data-cy="caption" mb={2} alignItems="flex-end">
             <P fontSize="LeadParagraph" fontWeight="bold">
-              {host.totalHostedCollectives}
+              {host.totalHostedCollectives ? host.totalHostedCollectives : 0}
             </P>
             <P ml={2} fontSize="Caption">
               {formatMessage(messages.collectives)}
             </P>
           </Flex>
           <Flex data-cy="caption" mb={2} alignItems="flex-end">
             <P fontSize="LeadParagraph" fontWeight="bold">
-              {host.members.nodes.length}
+              {host.members.totalCount ? host.members.totalCount : 0}
```

Comment:
We need to fetch `totalCount` in the GraphQL query. Anyway, as written elsewhere, this is not the right value for what we want to display. We can iterate later.